web3.js/src/programs/vote.ts

```diff
@@ -410,4 +410,24 @@ export class VoteProgram {
       data,
     });
   }
+
+  /**
+   * Generate a transaction to withdraw safely from a Vote account.
+   *
+   * This function was created as a safeguard for vote accounts running validators, `safeWithdraw`
+   * checks that the withdraw amount will not exceed the specified balance (in out case rent-exempt
+   * minimum). If you wish to close the vote account by withdrawing the full amount, call the
+   * `withdraw` method directly.
+   */
+  static safeWithdraw(
+    params: WithdrawFromVoteAccountParams,
+    balance: number,
```

Comment:
nit for total clarity
```suggestion
    currentVoteAccountBalance: number,
```